frontend/src/index.pug

```diff
@@ -37,10 +37,15 @@ html
           .timestamp-footer
             span Generated by &nbsp;
```

Comment:
if we not using additional buffer space at the side of "RepoSense" hyperlink word, can remove the additional space.

Reply:
> Can we style it like this?
> 
> > **RepoSense** (v1.2.3)

I think the Professor wants this additional buffer based on his last reply. So, I think can keep it.

Reply:
If that's the case, the buffer on the other end should be retained as well [on line 43 removed].

Reply:
I think have retained it. Check my latest commit?